src/plugins/intel_gpu/src/graph/graph_optimizer/concat_input_order.cpp

```diff
@@ -211,8 +211,12 @@ void concat_input_order::run(program& p) {
         auto mutable_prim = std::const_pointer_cast<concatenation>(prim);
         mutable_prim->input = new_input_info;
         // Correct users for shuffled features
-        for (auto& user : concat_node.get_users()) {
-            shuffle_features(*user, shuffled_ranges, p.get_stream());
+        try {
+            for (auto& user : concat_node.get_users()) {
```

Comment:
I don't get the motivation of this change. Why you just ignore some exception? I suppose if exception occurs here, then something went wrong and some part of the pass must be fixed.